lib/FilesHooks.php

```diff
@@ -461,8 +450,8 @@ protected function fileMoving($oldPath, $newPath) {
 			return;
 		}
 		$accessList = $this->getUserPathsFromPath($parentPath, $parentOwner);
-		$affectedUsers = $accessList['users'];
-		$oldUsers = $this->oldAccessList['users'];
+		$affectedUsers = $this->getAffectedUsers($accessList['users'], $fileId);
+		$oldUsers = $this->getAffectedUsers($this->oldAccessList['users'], $fileId);
```

Comment:
I think you need to do this where `$this->oldAccessList` is written, because `getMountsForFileId` and so on might not work now anymore. So you can not get people that had access before the action

Reply:
@nickvergessen  Thx for the feedback!
So pls correct me if i understood wrong: In this case, we can just leave this line as before:
$oldUsers = $this->oldAccessList['users'];
Because the new users will be notified (included in the previous line) and the old users, that loose the visibility to the file, don't need to be no notified. Correct?

Reply:
Search for:
```php
$this->oldAccessList = $this->getUserPathsFromPath($this->oldParentPath, $this->oldParentOwner);
```
And after add:
```php
$this->oldAccessList['users'] = $this->getAffectedUsers($this->oldAccessList['users'], $this->oldParentId);
```

Reply:
> Because the new users will be notified (included in the previous line) and the old users, that loose the visibility to the file, don't need to be no notified. Correct?

No, they should be notified that the file is deleted.

Reply:
@nickvergessen Great!! I got it now, make the adjusts and tested it! Perfect! Thx a lot!